test/cholesky/lkjcholesky.jl

```diff
@@ -181,6 +181,7 @@ using FiniteDifferences
         nkstests = 4 # use for appropriate Bonferroni correction for KS test
 
         @testset "rand" begin
+            @test rand(LKJCholesky(1, 0.5)) == Cholesky(ones(1, 1), 'U', 0)
```

Comment:
Sorry, forgot the default is L
```suggestion
            @test rand(LKJCholesky(1, 0.5)) == Cholesky(ones(1, 1), 'L', 0)
```

Reply:
Maybe the safer option would be
```suggestion
            @test rand(LKJCholesky(1, 0.5)).factors == ones(1, 1)
```
?